src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -271,7 +271,9 @@ class DocBaseViewer extends BaseViewer {
         // the file. Users without download permissions shouldn't be able to
         // interact with the text layer
         const canDownload = checkPermission(this.options.file, PERMISSION_DOWNLOAD);
-        this.findBar = new DocFindBar(this.findBarEl, this.findController, canDownload);
+        if (!this.getViewerOption('disableFindBar')) {
```

Comment:
I think it make sense to structure this like so we don't have a double negative
```
if (this.getViewerOption('disableFindBar') {
    return;
}
this.findBar = new DocFindBar(this.findBarEl, this.findController, canDownload);
```

Reply:
@pramodsum, sure I'll change it. It reads better too.